src/pretty.nit

```diff
@@ -223,6 +223,14 @@ class PrettyPrinterVisitor
 		if current_length == 0 and last_line_is_blank then return
 		previous_length = current_length
 		current_length = 0
+		if skip_empty then wait_addn += 1
+	end
+
+	# Force the line to end even is not skip_empty.
+	fun forcen do
```

Comment:
Forcen ? Strange name for a method, I don't understand its purpose either, event with the comment, care to explain ?